src/python/WMCore/MicroService/MSTransferor/DataStructs/Workflow.py

```diff
@@ -484,3 +484,25 @@ def getOpenRunningTimeout(self):
         :return: an integer with the amount of secs
         """
         return self.data.get("OpenRunningTimeout", 0)
+
+    def getInputData(self):
+        """
+        Returns all the primary and parent data that has to be locked
+        and transferred with Rucio
+        :return: a list of unique blocks
+        """
+        return self.getChunkBlocks()
```

Comment:
why do you need a method for doing a single return of a call to another method. I can understand that, though, if it is needed for other methods naming compatibility and following the same structure.